org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/filebased/NamespaceProperties.java

```diff
@@ -0,0 +1,98 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+package org.eclipse.winery.repository.backend.filebased;
+
+import java.io.Serializable;
+
+public class NamespaceProperties implements Serializable, Comparable<NamespaceProperties> {
+
+    private static final long serialVersionUID = -6867642303902116547L;
+
+    private String namespace;
+    private String prefix;
+    private String readableName = "";
+    private String upstreamRepository = "";
```

Comment:
add a comment explaining the purpose of the `upstreamRepository` field as it is not clear from the context of the pull requrest